set.mm

```diff
@@ -38138,21 +38163,50 @@ technically classes despite morally (and provably) being sets, like ` 1 `
       BCLM $.
   $}
 
+  ${
+    $d x y z ph $.  $d y z ch $.  $d y z ps $.
+    ss2abdv.1 $e |- ( ph -> ( ps -> ch ) ) $.
+    $( Deduction of abstraction subclass from implication.  (Contributed by NM,
+       29-Jul-2011.)  Avoid ~ ax-8 , ~ ax-10 , ~ ax-11 , ~ ax-12 .  (Revised by
+       Gino Giotto, 28-Jun-2024.) $)
+    ss2abdv $p |- ( ph -> { x | ps } C_ { x | ch } ) $=
+      ( vy vz cab cin wceq wss cv wcel wa df-in wsb df-clab bicomi sbequ wi weq
+      anbi12i abbii anbi12d sbievw ax-1 impd sbimdv ancld impbid syl5bb 3bitr4g
+      a1i eqrdv syl5eqr syl5eq df-ss sylibr ) ABDHZCDHZIZUSJUSUTKAVAFLZUSMZVBUT
+      MZNZFHZUSFUSUTOAVFBDFPZCDFPZNZFHZUSVIVEFVGVCVHVDVCVGBFDQRVDVHCFDQRUBUCAGV
+      JUSAVIFGPZBDGPZGLZVJMVMUSMVKVLCDGPZNZAVLVIVOFGFGUAVGVLVHVNBFGDSCFGDSUDUEA
+      VOVLAVLVNVLVLVNVLTTAVLVNUFUMUGAVLVNABCDGEUHUIUJUKVIGFQBGDQULUNUOUPUSUTUQU
+      R $.
+    $( $j usage 'ss2abdv' avoids 'ax-8' 'ax-10' 'ax-11' 'ax-12'; $)
+  $}
```

Comment:
this can probably be shortened by doing

```
ssrdv ( 3imtr4g (
  sbimdv (ss2abdv.1)
  df-clab
  df-clab
) )
```

Reply:
Theorem [ssrdv](https://us.metamath.org/mpeuni/ssrdv.html) uses ax-8 (while my proof doesn't).